src/main/java/drivers/DriverManager.java

```diff
@@ -0,0 +1,22 @@
+package drivers;
+
+import org.openqa.selenium.WebDriver;
+
+public class DriverManager {
+
+    private static ThreadLocal<WebDriver> webDriver = new ThreadLocal<>();
+
+    public static WebDriver getDriver() {
+        return webDriver.get();
+    }
+
+    public static void setWebDriver(WebDriver driver) {
+        webDriver.set(driver);
+    }
+
+    public static void quitDriver(WebDriver driver) {
+        if (null != driver) {
```

Comment:
Why do you compare null with driver not driver with null?

Reply:
fixed